### DISCUSSION TITLE
Loading dicoms with different planes

### DISCUSSION THREAD

**Discussion by ayvbrg** [TorchIO-project/torchio#632] [Q&A] [answered]
Hi, I am a newbie in medical imaging so please forgive me before hand for any silly errors. 

I have a series of dicoms belonging to a brain MRI. The dicoms are not in the same planes. Some are in sagittal while some are in axial. I can load them by specifying the directory in tio.ScalerImage and if the do .plot it shows me the different orientations of a slice. 

So my questions is does torchio automatically convert all the images into the same plane? If yes how does it select the plane to which it converts and if no is there a way i can achieve this.

**Comment by fepegar** [comment, marked as answer]
Hi, @benihime91.

> So my questions is does torchio automatically convert all the images into the same plane?

Images are loaded in the original orientation. The [`ToCanonical` transform](https://torchio.readthedocs.io/transforms/preprocessing.html#tocanonical) is used internally by `plot` so the orientation is consistent. It converts the images to the RAS+ orientation before plotting. For more information, you can take a look at its docs.

Does that make sense?
